pkg/log/zap/logger.go

```diff
@@ -34,7 +33,6 @@ func LoggerTo(destWriter io.Writer) logr.Logger {
 	syncer := zapcore.AddSync(destWriter)
 	conf := getConfig()
 
-	conf.encoder = &logf.KubeAwareEncoder{Encoder: conf.encoder, Verbose: conf.level.Level() < 0}
```

Comment:
Following it fixed made in the PR: https://github.com/operator-framework/operator-sdk/pull/1388/commits/f49484592eecb160259e2b162357c0982676fb92#diff-4407315726717f30063d558928ad9188R37

```go
zapf "sigs.k8s.io/controller-runtime/pkg/log/zap"
	
conf.encoder = &zapf.KubeAwareEncoder{Encoder: conf.encoder, Verbose: conf.level.Level() < 0}
`